examples/instruct_pix2pix/train_instruct_pix2pix.py

```diff
@@ -851,8 +851,6 @@ def collate_fn(examples):
         for step, batch in enumerate(train_dataloader):
             # Skip steps until we reach the resumed step
             if args.resume_from_checkpoint and epoch == first_epoch and step < resume_step:
-                if step % args.gradient_accumulation_steps == 0:
-                    progress_bar.update(1)
```

Comment:
Why are we removing this?

Reply:
Because we have set `initial=global_step` in the progress_bar so it will be consistent with `train_instruct_pix2pix_sdxl.py`'s progress_bar setup? As is proposed by bghira in [here](https://github.com/huggingface/diffusers/pull/9791#issuecomment-2444592784). If we don't remove this statement, the initial should be set to zero.